sdks/java/extensions/gcp-core/src/main/java/org/apache/beam/sdk/util/GcsUtil.java

```diff
@@ -436,8 +436,16 @@ public SeekableByteChannel open(GcsPath path)
    * @param type the type of object, eg "text/plain".
    * @return a Callable object that encloses the operation.
    */
-  public WritableByteChannel create(GcsPath path,
-      String type) throws IOException {
+  public WritableByteChannel create(GcsPath path, String type) throws IOException {
+    return create(path, type, uploadBufferSizeBytes);
+  }
+
+  /**
+   * Same as {@link GcsUtil#create(GcsPath, String)} but allows overriding
+   * {code uploadBufferSizeBytes}.
+   */
+  public WritableByteChannel create(GcsPath path, String type, Integer uploadBufferSizeBytes)
```

Comment:
Can't this take CreateOptions directly? That way the next option does not need to be plumbed.